### PR TITLE
Add release notes for Luna Streaming 2.8.0_1.0.0

### PR DESCRIPTION
**Pull request by eolivelli**

### CODE REVIEW

**Review by johnsmartco — CHANGES_REQUESTED** [review]
Enrico, thanks. Please put the section about Luna Streaming 2.8.0 (currently lines 48 to 100) above the section about LS 2.7.2 (currently lines 4 to 46).  

In the moved 2.8.0 section, add the date of that release (similar to what was done for 2.7.2 on L5).  

Please ping me when that is done, and I'll approve. If I have permissions in this GH, I'll then squash/merge. Thanks Enrico.

File: Luna_Streaming_2.8_ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+```
+chown -R 10000:10001 /pulsar/data
+```
+
+If you are are using the Luna Streaming Helm chart, you can enable automatic repair of the permissions using the `fixRootlessPermissions` setting. For more details on this setting, go [here](https://github.com/datastax/pulsar-helm-chart).
+
+## Luna Streaming Distribution 2.8.0 1.0.0
+
+This release is based on the [Apache Pulsar 2.8.2 release](https://pulsar.apache.org/release-notes/#280-mdash-2021-06-12-a-id280a). In addition to the contents of that release, it includes the following notable commits:
```
Comment (lhotari) [inline Luna_Streaming_2.8_ReleaseNotes.md:50]: 2.8.2 -> 2.8.0

**Review by johnsmartco — CHANGES_REQUESTED** [review]
Actually Enrico, disregard my prior comment. Instead, please do this:

L4:  change 2.7.2 to 2.8.0
L7:  change 2.7.2 to 2.8.0 
L36:  change 2.7.2 to 2.8.0 
L38: fix typos and change: "As Luna Streming 2.7 servies" to "As Luna Streaming 2.8 services"

The rest should be okay.

**Review by johnsmartco — APPROVED** [review]
Looks good.

File: Luna_Streaming_2.8_ReleaseNotes.md (hunk 1 above)
Comment (lhotari) [reply Luna_Streaming_2.8_ReleaseNotes.md:50]: @eolivelli @johnsmartco This typo remains in the final version.

Comment (lhotari) [reply Luna_Streaming_2.8_ReleaseNotes.md:50]: resolved by #20

**Pull request merged**